osf_tests/test_node.py

```diff
@@ -1,17 +1,24 @@
 import datetime
+import json
 
-from django.utils import timezone
 import mock
 import pytest
 import pytz
 import random
 import string
+# Python 3.x incompatible, use import builtins instead
+import __builtin__ as builtins
```

Comment:
It does work with mock.patch but it requires modifying osf/utils/migrations.py::ensure_licenses() to explicitly use `builtins.open` because I think it defaults to the python2 `__builtin__.open`. While the explicit `builtins.open` won't be necessary when we transition to python3, I think it's worth it just so that test_node isn't dependent on what will be an outdated import. I'll change this now.